docs/content/node-operation/access-node-setup.mdx

```diff
@@ -1,22 +1,327 @@
+<nil> DBG will generate networking key
+<nil> INF generated networking key
+<nil> DBG will generate staking key
+<nil> INF generated staking key
+<nil> DBG will generate db encryption key
+<nil> INF generated db encryption key
+<nil> DBG assembling node information address=flowaccess.mycompany.com:3569
+<nil> DBG encoded public staking and network keys networkPubKey=f493a74704f6961ae7903e062ecd58d990672858eff99aece7bfbccf3aa02de8f1a624ecbf21a01e8b2f4a5854c231fbe218edd7762a34fea881f3958a215305 stakingPubKey=ae8dcf81f3a70d72036b7ba2c586ed37ed0eb82b9c0a4aab998a8420f98894f94c14f84fa716e93654d3940fc0c8ff4d19b504c90a5b4918b28f421e9d3659dc2b7e246025ebeffea0d83cceefe315d7ed346dbe412fdac51b64997d97d29f7e
+<nil> INF wrote file bootstrap/public-root-information/node-id
+<nil> INF wrote file bootstrap/private-root-information/private-node-info_e737ec6efbd26ef43bf676911cdc5a11ba15fc6562d05413e6589fccdd6c06d5/node-info.priv.json
+<nil> INF wrote file bootstrap/private-root-information/private-node-info_e737ec6efbd26ef43bf676911cdc5a11ba15fc6562d05413e6589fccdd6c06d5/secretsdb-key
+<nil> INF wrote file bootstrap/public-root-information/node-info.pub.e737ec6efbd26ef43bf676911cdc5a11ba15fc6562d05413e6589fccdd6c06d5.json
+
+$tree ./bootstrap/
+./bootstrap/
+├── private-root-information
+│        └── private-node-info_e737ec6efbd26ef43bf676911cdc5a11ba15fc6562d05413e6589fccdd6c06d5
+│        ├── node-info.priv.json
+│        └── secretsdb-key
+└── public-root-information
+    ├── node-id
+    └── node-info.pub.e737ec6efbd26ef43bf676911cdc5a11ba15fc6562d05413e6589fccdd6c06d5.json
+
+3 directories, 4 files
+```
+
+> ⚠️ _Use a fully qualified domain name for the network address. Please also include the port number in the network address e.g. `flowaccess.mycompany.com:3569`_
+
+> ⚠️ _Do not include in `http://` in the network address._
+
+> If you would like to stake multiple access nodes, please ensure you generate a unique identity for each node.
+
+Your node identity has now been generated. Your <b>node ID</b> can be found in the file `./bootstrap/public-root-information/node-id`.
+
+```shell:title=Example
+$cat ./bootstrap/public-root-information/node-id
+e737ec6efbd26ef43bf676911cdc5a11ba15fc6562d05413e6589fccdd6c06d5
+```
+
+> All your private keys should be in the `bootstrap` folder created earlier. Please take a back up of the entire folder.
+
+## Step  2 - Stake the node
+
+You need to now register the node on chain by staking the node via [Flow Port](https://port.onflow.org/).
+
+[Here](https://developers.flow.com/nodes/flow-port/staking-guide) is a guide on how to use Flow port if you are not familiar with it.
+If you are staking via a custody provider or would like to directly submit a staking transaction instead follow this [guide](/nodes/staking#how-do-i-stake).
+
+Fund you Flow account with at least 100.01 FLOW tokens, which covers the required stake plus the storage deposit.
```

Comment:
This can be 100.00. it is `>=` in the contract

Reply:
when testing on Flow port, I funded the account with 100FLOW and the transaction failed with an insufficient funds error since it needed to keep some FLOW in the account for storage.

Reply:
oh, I read that wrong. I thought you meant register the node. We're all good. :)